model/src/schema.js

```diff
@@ -147,7 +147,8 @@ const schema = joi.object().required().keys({
   declaration: joi.string().allow('').optional(),
   outputs: joi.array().items(outputSchema),
   payApiKey: joi.string().allow('').optional(),
-  skipSummary: joi.boolean().default(false)
+  skipSummary: joi.boolean().default(false),
+  version: joi.number().default(0)
```

Comment:
should we default to version in package.json?

Reply:
It's an interesting thought. I don't think so though as I imagine we will frequently make changes to the package.json version which don't involve changes to the underlying schema. This was intended to help us know whether we need to migrate the schema before using it in the runner / designer